src/test/java/com/requestrealpiano/songrequest/domain/letter/LetterRepositoryTest.java

```diff
@@ -1,38 +1,78 @@
 package com.requestrealpiano.songrequest.domain.letter;
 
+import com.requestrealpiano.songrequest.domain.letter.request.PaginationParameters;
 import com.requestrealpiano.songrequest.testconfig.BaseRepositoryTest;
-import com.requestrealpiano.songrequest.testobject.LetterFactory;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.domain.Sort.Direction;
 
 import java.time.LocalDateTime;
+import java.util.ArrayList;
 import java.util.List;
-import java.util.Optional;
+import java.util.stream.Stream;
 
+import static com.requestrealpiano.songrequest.domain.letter.RequestStatus.*;
+import static com.requestrealpiano.songrequest.global.constant.SortProperties.CREATED_DATE_TIME;
 import static com.requestrealpiano.songrequest.testobject.LetterFactory.createLetter;
 import static com.requestrealpiano.songrequest.testobject.LetterFactory.createLetters;
+import static com.requestrealpiano.songrequest.testobject.PaginationFactory.createPageRequest;
+import static com.requestrealpiano.songrequest.testobject.PaginationFactory.createPaginationParametersOf;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
 
 class LetterRepositoryTest extends BaseRepositoryTest {
 
     @Autowired
     LetterRepository letterRepository;
 
+    @ParameterizedTest
+    @MethodSource("paginationFindAllLettersParameters")
+    @DisplayName("Paging 처리된 Letter 목록 조회 테스트")
+    void pagination_find_all_letters(List<Letter> letters, Integer page, Integer size) {
+        // given
+        PaginationParameters parameters = createPaginationParametersOf(page, size);
+        PageRequest pageRequest = PageRequest.of(parameters.getPage(), parameters.getSize(),  Sort.by(Direction.DESC, CREATED_DATE_TIME.getFieldName()));
+
+        // when
+        letterRepository.saveAll(letters);
+        Page<Letter> savedLetters = letterRepository.findAll(pageRequest);
+
+        // then
+        assertThat(savedLetters.getContent().size()).isEqualTo(size);
+        assertThat(savedLetters.getTotalElements()).isEqualTo(letters.size());
```

Comment:
using recursive comparison()도 한번 봐주셨으면 합니다. 객체안의 필드가 또 객체일 경우에는 저 메서드를 사용하여 안에 있는 객체도 전부 검사하거든요. 이거추가했을 때 통과가 안되는 테스트도 있으니 꼭 확인하시길 바랍니다. 통과가 안됐다? -> 그럼 잘못 만든 비즈니스 코드일 가능성있음
```suggestion
        assertThat(savedLetters.getContent().size()).usingRecursiveComparison().isEqualTo(size);
```

Reply:
몰랐던 내용인데 공유 해주셔서 감사합니다!